classes/05class/exercises/c05-3m01/obrientimothya/docker-compose.yml

```
@@ -0,0 +1,10 @@
+---
+version: "3"
+services:
+  aws:
+    image: obrientimothya/aws-cli:2.0.0
```

For avoiding the extra `build` target, you can use
`    
build: .




```suggestion
    image: .
```

For avoiding the extra `build` step in your Makefile you can use `build` instead of `image` in docker-compose for temporary builds. The image is built and cached locally